### ISSUE TITLE
AZ 204 Lab12 Docker Image

### ISSUE THREAD

**Issue by knightie** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#539]
# Module: AZ 204
#### Step:  Step 5

Description of issue
You will still see the same previos error pages.

Repro steps:
Just follow the steps and you will not see the images nor the video being displayed.
Sorry that I can not find your source code to see how you take in and render the environment variable, else can fix it for you.
1. I have checked the links generated from the storage, they work well, the path is correct too, both for pictures and the video.
2. I have also checked the links given by the CDN, they work well too.
3. I have also checked the web app Environment variables as instructed.

But since I can not access to your source code, I don't know how you path the images and the video.

**Comment by TesterTesterson1004** [comment]
I am experiencing the same behavior and issue in a fresh instance of this lab.  

I wonder if it might have something to do with this - possibly typo? :  When I used 'microsoftlearningedx-html-landinpage:latest' the issue repros.  On the other hand, if I start a new/fresh instance and use 'microsoftlearning/edx-html-landing-page:latest' I don't see the issue.

At least, I was able to proceed further.  I can currently see the CDN network successfully hosting the welcome.mp4 content, although none of the .jpg files appear to have been initialized/replicated as-yet.  The note indicates it can take from 5-15 minutes, and it has been 20 mins now....uncertain if this means there is yet another issue, or the CDN is just taking a looooonnnggg time past the estimated.
